src/services/appDataService.ts

```diff
@@ -0,0 +1,112 @@
+import taggingUtils, { tagKeys } from '../lib/taggingUtils';
+import documentRoutes from '../routes/documentRoutes';
+import recordService from './recordService';
+import { prepareSearchParameters } from './fhirService';
+
+export interface DecryptedAppData {
+  id?: string;
+  data: any;
+  tags?: string[];
+  customCreationDate?: Date;
+  updatedDate?: Date;
+  commonKeyId?: string;
+}
+
+interface IRecord {
```

Comment:
My linter always complains about the I in front of interfaces. Don't know what the best approach with naming of interfaces is though.
![Screenshot 2020-06-30 at 19 03 49](https://user-images.githubusercontent.com/13232878/86155430-b2aa7400-bb04-11ea-9ee5-f779773e5541.png)

Reply:
Oh yeah, this seems to be something we should correct for all of the repository (though I'd wager it's outside the scope of this PR), see the discussion here https://github.com/typescript-eslint/typescript-eslint/issues/374